typing/ctype.ml

```diff
@@ -2180,80 +2226,12 @@ let rec estimate_type_jkind ~expand_component env ty =
   | Tpoly (ty, _) -> estimate_type_jkind ~expand_component env ty
```

Comment:
As discussed offline, this allows type variables to escape their scope. For now, we can `Jkind.round_up`, and in the future we'll want something like `round_up_to_remove_these_variables`